lib/Form/Templates/FormTemplate.php

```diff
@@ -39,6 +39,10 @@ public function getStyles(): string {
   margin-bottom: 20px;
 }
 
+.last .mailpoet_paragraph {
```

Comment:
I think it would be nicer to have it in one place here https://github.com/mailpoet/mailpoet/blob/master/lib/Form/Util/Styles.php#L116